sw/device/lib/aes.c

```diff
@@ -0,0 +1,116 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "aes.h"
+
+#include "aes_regs.h"
+#include "common.h"
```

Comment:
Use absolute imports. Note sure why this is building...

Reply:
I switched to absolute imports, except for `aes_regs.h`. This is generated and I now marked it as such.